.github/workflows/msvc-ci.yml

```diff
@@ -0,0 +1,153 @@
+          
+          # Unzip
+          $gtk_dir = Join-Path -Path "${{ runner.temp }}" -ChildPath "gtk";
+          Expand-Archive -Path $gtk_zip_file -DestinationPath "${gtk_dir}";
+
+          # Some helper variables
+          $gtk_bin_dir = Join-Path -Path $gtk_dir -ChildPath "bin";
+          $gtk_pkg_config_dir = Join-Path -Path $gtk_dir -ChildPath "lib" |
+            Join-Path -ChildPath "pkgconfig";
+
+          # Set environment variable GTK_DIR pointing to GTK's directory
+          Add-Content "${{ github.env }}" "GTK_DIR=${gtk_dir}";
+                    
+          # Set environment variable GTK_BIN_DIR pointing to GTK's bin directory
+          Add-Content "${{ github.env }}" "GTK_BIN_DIR=${gtk_bin_dir}";
+
+          # Set environment variable GTK_PKG_CONFIG_PATH pointing to GTK's pkg-config directory
+          Add-Content "${{ github.env }}" "GTK_PKG_CONFIG_PATH=${gtk_pkg_config_dir}";
+          
+          # Place GTK bin directory on system PATH environment variable
+          Add-Content "${{ github.path }}" "${gtk_bin_dir}";
+      
+      - name: Download and extract LuaInstaller, and set an environment variable for it
+        run: |
+          $luainstaller_zip_file = Join-Path -Path "${{ runner.temp }}" -ChildPath "luainstaller.zip";
+
+          # Download
+          Invoke-WebRequest -Uri "${{ env.LUAINSTALLER_URL }}" -OutFile $luainstaller_zip_file;
+
+          # Unzip
+          Expand-Archive -Path $luainstaller_zip_file -DestinationPath "${{ runner.temp }}";
+
+          $luainstaller = Join-Path -Path "${{ runner.temp }}" -ChildPath "LuaInstaller.Console" |
+            Join-Path -ChildPath "LuaInstaller.Console.exe";
+
+          # Set LUA_INSTALLER environment variable pointing to the app
+          Add-Content "${{ github.env }}" "LUA_INSTALLER=${luainstaller}";
+      
+      - name: Install Lua ${{ matrix.version }} on GTK's directory and set environment variables for it
+        run: |
+          & "${{ env.LUA_INSTALLER }}" install "dest-dir=${{ env.GTK_DIR }}" "version=${{ matrix.version }}";
+
+          # Test Lua
+          & lua -v;
+
+          # Find Lua's pkgconfig (.pc) file
+          $lua_pc = Get-ChildItem "${{ env.GTK_PKG_CONFIG_PATH }}" -File |
+            Where-Object Name -Like "lua*.pc" |
+            Select-Object -ExpandProperty BaseName -First 1;
+
+          # Set LUA_PC environment variable pointing to Lua's (.pc) file
+          Add-Content "${{ github.env }}" "LUA_PC=${lua_pc}";
+
+      - name: Setup Python 3.12
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+      
+      - name: Install meson
+        run: pip install meson
+
+      - name: Setup MSVC dev-prompt
+        uses: ilammy/msvc-dev-cmd@v1
+
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Configure lgi through meson
+        run: meson setup lgi-build . --prefix "${{ env.GTK_DIR }}" "-Dlua-pc=${{ env.LUA_PC }}" -Dtests=false
```

Comment:
> I'm almost afraid to ask, but... what's up with `-Dtests=false`?

I don't know why, but I keep getting a Python-related error when setting -Dtests=true (some kind of _winapi.CreateProcess error), which I don't know how to solve (probably it has to build Python from sources, then build the whole GTK stack). I think building the whole GTK stack on Windows, even through wingtk project, is a bit of non-sense, because it certainly takes 30 minutes or longer.